<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuSecre</class>
 <widget class="QMainWindow" name="MenuSecre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secretaria</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QWidget" name="widget" native="true">
         <property name="maximumSize">
          <size>
           <width>322</width>
           <height>522</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="0">
           <widget class="QLineEdit" name="fPaciente">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Folio Paciente</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="folio">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Folio</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDateEdit" name="fecha">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximumDate">
             <date>
              <year>2100</year>
              <month>12</month>
              <day>31</day>
             </date>
            </property>
            <property name="minimumDate">
             <date>
              <year>2019</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="Agregar">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Agregar</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLineEdit" name="uPsicologo">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Psicologo</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QTimeEdit" name="hora">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <widget class="QPushButton" name="Mostrar">
            <property name="maximumSize">
             <size>
              <width>209</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Mostrar</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QTableView" name="tableView">
            <property name="minimumSize">
             <size>
              <width>418</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="Eliminar">
            <property name="maximumSize">
             <size>
              <width>209</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="eFolio">
            <property name="maximumSize">
             <size>
              <width>209</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Folio a eliminar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>folio</tabstop>
  <tabstop>fecha</tabstop>
  <tabstop>hora</tabstop>
  <tabstop>fPaciente</tabstop>
  <tabstop>uPsicologo</tabstop>
  <tabstop>Agregar</tabstop>
  <tabstop>eFolio</tabstop>
  <tabstop>Mostrar</tabstop>
  <tabstop>Eliminar</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
